build/test-scripts/ios-uitest-run.sh

```diff
@@ -238,7 +238,7 @@ else
 	echo "  Test filters: $UNO_TESTS_FILTER"
 
 	## Run tests
-	dotnet run -c Release -- --results-directory $UNO_ORIGINAL_TEST_RESULTS_DIRECTORY --settings .runsettings --filter "$UNO_TESTS_FILTER" || true
+	dotnet run -c Release -- --results-directory $UNO_ORIGINAL_TEST_RESULTS_DIRECTORY --hangdump --hangdump-timeout $UITEST_TEST_TIMEOUT -⁠-⁠hangdump-⁠filename $LOG_FILEPATH/hang.dump --settings .runsettings --filter "$UNO_TESTS_FILTER" || true
```

Comment:
FYI https://github.com/dotnet/docs/pull/45577
Not sure if this will fail or not as those were copied

Reply:
It was copied :) I've removed them